packages/tupaia-web/src/views/ForgotPasswordModal.tsx

```diff
@@ -32,16 +39,28 @@ export const CancelButton = styled(RouterButton).attrs({
   border-color: ${({ theme }) => theme.palette.text.secondary};
   margin-top: 1.3rem;
 `;
+
 const CheckEmailMessage = styled.p`
   text-align: center;
   padding: 0 0.9375rem;
 `;
 
+const LinkText = styled(Typography)`
+  font-size: 11px;
+  line-height: 15px;
+  color: white;
+
+  a {
+    color: white;
+  }
+
+  ${CancelButton} + & {
+    margin-top: 1.3rem;
+  }
+`;
+
 export const ForgotPasswordModal = () => {
-  const formContext = useForm({
-    mode: 'onChange',
```

Comment:
This failed testing because the onChange mode means that the form starts validating as soon as a change is made and then it displays validation errors too soon.